ibm_catalog.json

```diff
@@ -234,11 +209,11 @@
               "key": "private_key_format",
               "options": [
                 {
-                  "displayname": "der",
+                  "displayname": "Der",
```

Comment:
Please use the correct display names. For example here it should be DER. This can be easily seen upon creation of root certificate CA

<img width="326" height="267" alt="image" src="https://github.com/user-attachments/assets/90537f68-5a73-4ae9-9942-4bbf46d4b303" />